core/src/main/java/org/hisp/dhis/android/network/attribute/AttributeService.kt

```diff
@@ -26,22 +26,21 @@
  *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package org.hisp.dhis.android.core.attribute.internal
+package org.hisp.dhis.android.network.attribute
 
-import org.hisp.dhis.android.core.arch.api.HttpServiceClient
 import org.hisp.dhis.android.core.arch.api.fields.internal.Fields
 import org.hisp.dhis.android.core.arch.api.filters.internal.Filter
-import org.hisp.dhis.android.core.arch.api.payload.internal.PayloadJackson
 import org.hisp.dhis.android.core.attribute.Attribute
+import org.hisp.dhis.android.network.common.HttpServiceClientKotlinx
 import org.koin.core.annotation.Singleton
 
 @Singleton
-internal class AttributeService(private val client: HttpServiceClient) {
-    suspend fun getAttributes(
+internal class AttributeService(private val client: HttpServiceClientKotlinx) {
+    suspend fun attributes(
```

Comment:
Why removing the "get" from this method? In this case there is only a get request, but other times there will be post or delete

Reply:
Yeah, I was just following what you did in `OptionService` to keep things consistent, but I did think while working on it that this might not be the clearest way. 

https://github.com/dhis2/dhis2-android-sdk/pull/2253/files#diff-d94cb9660272a03d5e53c7032f510f66a9468204fc04d092d831ef9099f18800